nautobot/docs/installation/region-and-site-data-migration-guide.md

```diff
@@ -0,0 +1,405 @@
+    "device__site__region__id": "0ab47314-123144-2132f6-b964-9ekmwkq12c48ce0",
+    "device__site__region__id__in": ["0ab47314-123144-2132f6-b964-9ekmwkq12c48ce0", "kml21345d4-6e123-471e-8as1231-27f25lmsahca308f5"],
+    "device__site__region__parent__name": "North America",
+    "device__site__region__parent__name__in": ["North America", "Europe", "South America"],
+    "device__site__region__parent__slug": "north-america",
+    "device__site__region__parent__slug__in": ["north-america", "europe", "south-america"],
+    "device__site__region__parent__id": "0kmskqjneq14-945840-2132f6-b964-9ekmwkq12c48ce0",
+    "device__site__region__parent__id__in": ["0kmskqjneq14-945840-2132f6-b964-9ekmwkq12c48ce0", "qwe0kmjn1lohg-12olm-kmlie-8as1231-km2l1321isdan234f5"],
+}
+```
+
+To modify the data correctly, we need to replace query filter strings to those of model class `Location`:
+    1. Replace all occurrences of "site" with "location" in the **Key** portion (before ":") of the data **not Value** (after ":").
+    2. Replace all occurrences of "region" with "parent" in the **Key** portion (before ":") of the data **not Value** (after ":").
+    3. Since we are keeping the uuids of the newly created "Site"/"Region" type locations the same from those of the old `Site`/`Region` instances, we do not need to change the uuid values in "..__id" and "..__id__in" Keys.
+
+The updated JSON data might look like this:
+
+```json
+{
+    "device__location__name": "AMS01",
+    "device__location__name__in": ["AMS01", "ATL01", "ETX02"],
+    "device__location__slug": "ams01",
+    "device__location__slug__in": ["ams01", "atl01", "etx02"],
+    "device__location__id": "0ab47314-2944-45f6-b964-9e009fc48ce0",
+    "device__location__id__in": ["0ab47314-2944-45f6-b964-9e009fc48ce0", "b09545d4-6e2b-471e-8f07-27f25ca308f5"],
+    "device__location__parent__name": "United States",
+    "device__location__parent__name__in": ["United States", "United Kingdom", "Greece"],
+    "device__location__parent__slug": "united-states",
+    "device__location__parent__slug__in": ["united-states", "united-kingdom", "greece"],
+    "device__location__parent__id": "0ab47314-123144-2132f6-b964-9ekmwkq12c48ce0",
+    "device__location__parent__id__in": ["0ab47314-123144-2132f6-b964-9ekmwkq12c48ce0", "kml21345d4-6e123-471e-8as1231-27f25lmsahca308f5"],
+    "device__location__parent__parent__name": "North America",
+    "device__location__parent__parent__name__in": ["North America", "Europe", "South America"],
+    "device__location__parent__parent__slug": "north-america",
+    "device__location__parent__parent__slug__in": ["north-america", "europe", "south-america"],
+    "device__location__parent__parent__id": "0kmskqjneq14-945840-2132f6-b964-9ekmwkq12c48ce0",
+    "device__location__parent__parent__id__in": ["0kmskqjneq14-945840-2132f6-b964-9ekmwkq12c48ce0", "qwe0kmjn1lohg-12olm-kmlie-8as1231-km2l1321isdan234f5"],
+}
+```
+
+### Other Data Model Specific ObjectPermission e.g. ObjectChange
```

Comment:
Use of ObjectChange as a permissions example here feels a bit contrived to me. If we're looking for one with a content-type reference and/or a generic foreign key, maybe something like ExportTemplate might be a better example?

Reply:
It seems like `owner_content_type` and `owner_object_id` only involve "extras.gitrepository" and `GitRepository` uuids. I might opt for `Note` model as an example.